website/src/data/users.tsx

```diff
@@ -748,7 +748,7 @@ const Users: User[] = [
   },
   {
     title: 'GeekyWeb',
-    description: 'Global State and Logic Framework for reactive Applications',
+    description: 'Best documentation website for all developers and programmers.',
```

Comment:
Sorry I didn't review your description before🤦‍♂️ However, this description is not very descriptive. Please be more value-neutral, and describe what's included on your site, not just that "it's the best".